### PR TITLE
NEUBURG-1: Reduce overhead and integrate npm package

### PR DESCRIPTION
**Pull request by michael-markl**
This pull request belongs to an issue on our [bugtracker](https://integreat.atlassian.net/). 
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **NEUBURG**.

This PR reduces all the overhead we don't need in this project and adds the integreation of the npm package with the shared code.

### CODE REVIEW

File: src/modules/endpoint/endpoints/living.js [hunk 1]
```diff
@@ -3,32 +3,41 @@ import EndpointBuilder from '../EndpointBuilder'
...
+  tempUrl.shift()
+  return tempUrl.join('/')
+}
+
+const cityConfig = getCurrentCityConfig()
 
 export default new EndpointBuilder('living')
-  .withStateToUrlMapper(state => `https://cms.integreat-app.de/${state.router.params.location}` +
-    `/de/wp-json/extensions/v0/modified_content/pages?since=1970-01-01T00:00:00Z`)
+  .withStateToUrlMapper(state => `https://cms.integreat-app.de/${cityConfig.cmsName}/de/wp-json/extensions/v0/modified_content/pages?since=1970-01-01T00:00:00Z`)
```
Comment (NKlug) [inline src/modules/endpoint/endpoints/living.js:17]: We should update this to Api v3 eventually.

File: src/modules/endpoint/endpoints/locations.js [hunk 2]
```diff
@@ -23,7 +23,7 @@ export default new EndpointBuilder('locations')
         code: stripSlashes(location.path),
         live: location.live,
         eventsEnabled: location['ige-evts'] === '1',
-        extrasEnabled: true
+        extrasEnabled: true // todo
```
Comment (NKlug) [inline src/modules/endpoint/endpoints/locations.js:26]: Is there already a task for this?

**Review by NKlug — COMMENTED** [review]
Very neat all in all :+1:

File: src/modules/endpoint/endpoints/locations.js (hunk 2 above)
Comment (michael-markl) [reply src/modules/endpoint/endpoints/locations.js:26]: Hmm... nope, I'm not sure where this todo came from, but its irrelevant for us...
We could strip the locationsEndpoint I think...

File: src/modules/endpoint/endpoints/living.js (hunk 1 above)
Comment (michael-markl) [reply src/modules/endpoint/endpoints/living.js:17]: yup, but no prio

Comment (NKlug) [reply src/modules/endpoint/endpoints/living.js:17]: I created a task :)

File: src/modules/endpoint/endpoints/locations.js (hunk 2 above)
Comment (NKlug) [reply src/modules/endpoint/endpoints/locations.js:26]: Good idea, do you want to do it in this PR?

Comment (michael-markl) [reply src/modules/endpoint/endpoints/locations.js:26]: No, not really :D

**Review by NKlug — APPROVED** [review]

File: src/modules/endpoint/endpoints/locations.js (hunk 2 above)
Comment (steffenkleinle) [reply src/modules/endpoint/endpoints/locations.js:26]: this becomes obsolete with the use of the api v3 of the locations (cities) endpoint, which was already done in the WebApp, which isn't lots of work.

File: src/modules/endpoint/endpoints/living.js (hunk 1 above)
Comment (steffenkleinle) [reply src/modules/endpoint/endpoints/living.js:17]: Would be nice, yet we have to wait for https://github.com/Integreat/cms/issues/708 to have relative urls. The cities endpoint could be updated already.

File: src/modules/app/containers/I18nProvider.js [hunk 3]
```diff
@@ -1,14 +1,12 @@
 import i18n from 'i18next'
```
Comment (steffenkleinle) [inline src/modules/app/containers/I18nProvider.js:1]: Do we need the I18nProvider at all? If I can remember it right, the Wohnungsbörse should only be available in german

File: src/modules/endpoint/endpoints/locations.js (hunk 2 above)
Comment (steffenkleinle) [reply src/modules/endpoint/endpoints/locations.js:26]: is this even relevant for the Wohnungsbörse whether extras (and events) are enabled or not?

File: src/modules/endpoint/reducers/index.js [hunk 4]
```diff
@@ -1,20 +1,14 @@
...
  * Contains all reducers from all endpoints which are defined in {@link './endpoints/'}
  */
 const endpoints = [
   livingEndpoint,
-  languagesEndpoint,
-  locationEndpoint,
-  categoriesEndpoint,
   disclaimerEndpoint,
-  eventsEndpoint
+  locationEndpoint
```
Comment (steffenkleinle) [inline src/modules/endpoint/reducers/index.js:11]: location_s

File: src/modules/app/containers/I18nProvider.js (hunk 3 above)
Comment (michael-markl) [reply src/modules/app/containers/I18nProvider.js:1]: Well, it's better to leave it there, so we can handle all ui-text-related stuff in the locales.json.

Comment (steffenkleinle) [reply src/modules/app/containers/I18nProvider.js:1]: okay :+1:

File: src/modules/endpoint/endpoints/living.js (hunk 1 above)
Comment (NKlug) [reply src/modules/endpoint/endpoints/living.js:17]: Seems like this issue was already closed a few days ago.

File: src/modules/city-detection/cityConfigs.js [hunk 5]
```diff
@@ -0,0 +1,11 @@
+import CityConfig from './CityConfig'
```
Comment (maxammann) [inline src/modules/city-detection/cityConfigs.js:1]: I'd rather have this directly in `getCurrentCityConfig.js`

**Pull request merged**